src/catalog/catalog.cpp

```diff
@@ -0,0 +1,159 @@
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include "catalog/catalog.h"
+#include "catalog/database_handle.h"
+#include "loggers/catalog_logger.h"
+#include "storage/storage_defs.h"
+#include "transaction/transaction_manager.h"
+
+namespace terrier::catalog {
+
+std::shared_ptr<Catalog> terrier_catalog;
+
+Catalog::Catalog(transaction::TransactionManager *txn_manager) : txn_manager_(txn_manager) {
+  CATALOG_LOG_INFO("Creating catalog ...");
```

Comment:
You should go through the LOG calls and adjust them. 
INFO should be used for messages that are output under normal operation. There should be very little.
DEBUG should be used for logging additional information when the code goes through some error path and you want to record what is happening.
TRACE is for tracing execution flow. Most of the current messages are probably TRACE level.

Reply:
Changed.